castanet/counter.py

```diff
@@ -179,7 +222,15 @@ def match_class_defs(cast_dict):
 
 
 def count_class_defs_without_docstrings(class_count: Dict) -> int:
-    """Count the number of class definitions without docstrings."""
+    """Count the number of class definitions without docstrings.
+
+    Args:
+        class_count (Dict): function and docstring counts per file
+
+    Returns:
+        int: class_total - docstring_total
```

Comment:
Rather than repeating what the return statement says, you should say what the variable names mean. In this case I would put something like `int: total number of classes - total number of docstrings`